src/bundle/Resources/public/js/scripts/fieldType/base/base-rich-text.js

```diff
@@ -263,38 +264,50 @@
             const counterWrapper = container.parentElement.querySelector('.ez-character-counter');
 
             if (counterWrapper) {
-                const countableTags = this.getCountableTags(editorHtml);
                 const wordWrapper = counterWrapper.querySelector('.ez-character-counter__word-count');
                 const charactersWrapper = counterWrapper.querySelector('.ez-character-counter__character-count');
-                let characterCount = 0;
-                let wordCount = 0;
+                const words = this.getTextNodeValues(editorHtml);
 
-                countableTags.forEach((tag) => {
-                    const sanitizedText = this.cleanWhiteCharacters(tag.innerText);
+                wordWrapper.innerText = words.length;
+                charactersWrapper.innerText = words.join(' ').length;
```

Comment:
It seems that, if two words are separated by more than 1 space we count only first space as character. Does it work like this? is this intended?

Reply:
That's true, this case will be handled separately.